icevision/models/interpretation.py

```diff
@@ -236,14 +235,19 @@ def plot_top_losses(
             ann2 = "\n".join(ann[4:])
             anns.append((ann1, ann2))
 
+        # reload only the displayed images
+        displayed_preds = [
+            Prediction(pred=p, ground_truth=s.load())
```

Comment:
Do we have the problem here where the predictions and even the labels in the ground truth will be related to the transformed img but `s.load()` will load the original img?

Reply:
just confirmed that we do =x

Reply:
yeah this is my suboptimal solution to this general problem